app/assets/javascripts/app/views/hovercard_view.js

```diff
@@ -140,6 +143,14 @@ app.views.Hovercard = app.views.Base.extend({
       top: p_pos.top + p_height - 25,
       left: p_pos.left
     });
-  }
+  },
+  
+  mouseIsOverElement: function(element, event) {
+  	var el_pos = element.offset();
```

Comment:
We use two spaces for indentation I think, @svbergerem do you know why Hound isn't complaining about this?